lib/Backend/GlobOpt.cpp

```diff
@@ -9585,6 +9584,32 @@ GlobOpt::TypeSpecializeInlineBuiltInBinary(IR::Instr **pInstr, Value *src1Val, V
             break;
         }
 
+        case Js::OpCode::InlineMathPow:
+        {
+#ifndef _M_ARM
```

Comment:
Will consider to apply the same optimization for arm later.

---
In reply to: [57507236](https://github.com/Microsoft/ChakraCore/pull/644#discussion_r57507236) [](ancestors = 57507236)